src/styles/table/table.css

```diff
@@ -0,0 +1,40 @@
+@import url(../../themes/themes.css);
+/*
+ *  Styling for native-HTML tables.
+ **/
+
+table {
+  border-collapse: collapse;
+}
+
+tr {
+  background-color: var(--surface);
+  border-bottom: 1px solid var(--surface-2dp);
+}
+
+tr:hover {
+  background-color: var(--surface-1dp);
+}
+
+tr:hover tr:hover {
```

Comment:
hover hover?

Reply:
I don't think I understand the question.  This rule ensures that we get the hover behavior for (singly) nested tables.

Reply:
... but per below, I didn't end up needing this.